docs/install.md

```diff
@@ -442,6 +442,11 @@ features](#alpha-features) to be used.
   name, kind, and API version information for each `TaskRun` and `Run` in the `PipelineRun` instead. Set it to "both" to
   do both. For more information, see [Configuring usage of `TaskRun` and `Run` embedded statuses](pipelineruns.md#configuring-usage-of-taskrun-and-run-embedded-statuses).
 
+- `enable-provenance-in-status`: set this flag to "true" to enable recording 
+  the `provenance` field in `TaskRun` and `PipelineRun` status. The `provenance` field contains 
+  the key authenticated metadata about how a software artifact was built i.e. the source
+  where remote resource came from.
+
```

Comment:
we should also list this feature among the alpha features with its individual feature flag - https://github.com/tektoncd/pipeline/blob/main/docs/install.md#alpha-features

Reply:
Thanks @jerop ! Good idea. I've added this to the table. PTAL!

Reply:
I've also updated the paragraph above the alpha features table. PTAL. Thank you very much!

cc @dibyom